service/rag_processor.py

```
@@ -0,0 +1,212 @@
+            
+            if chunks_with_metadata:
+                self.vector_store = FAISS.from_texts(
+                    texts=[c["content"] for c in chunks_with_metadata],
+                    embedding=self.embeddings,
+                    metadatas=[c["metadata"] for c in chunks_with_metadata]
+                )
+
+            try:
+                from langdetect import detect
+                detected_language = detect(text[:1000])  # Ограничиваем текст для определения языка
+            except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)